app/src/main/java/org/listenbrainz/android/ui/screens/onboarding/permissions/PermissionScreens.kt

```diff
@@ -236,6 +258,37 @@ fun PermissionCard(
     }
 }
 
+@Composable
+fun CrashReportingToggle(viewModel: DashBoardViewModel = hiltViewModel()) {
```

Comment:
Please re-use existing components and do not inject view-model into preview able composable functions. When you inject a hilt view-model (dependency injected class) into a composable function, the preview window is unable to create instance of the view-model because some classes are too complex to be instantiated like DBs or contexts and builds the preview in an isolated sandbox. You can however instantiate view-models in cases where you do not want previews.